lib/src/ui.dart

```diff
@@ -151,17 +229,14 @@ class NetworkLoggerScreen extends StatelessWidget {
     );
   }
 
-  final TextEditingController searchController =
-      TextEditingController(text: null);
+  final TextEditingController searchController = TextEditingController(text: null);
 
   /// filte events with search keyword
   List<NetworkEvent> getEvents() {
     if (searchController.text.isEmpty) return eventList.events;
 
     final query = searchController.text.toLowerCase();
-    return eventList.events
-        .where((it) => it.request?.uri.toLowerCase().contains(query) ?? false)
-        .toList();
+    return eventList.events.where((it) => it.request?.uri.toLowerCase().contains(query) ?? false).toList();
```

Comment:
Again not important. No need to make any changes. I just would like to recommend to not make this kind of decorative changes. Yeah It's not a big deal, but some projects might not accept PR with this kind of changes since they don't provide any real value and usually this kind of decisions are subjective.

PS: keep it as is, i don't care about it but just wanted to add quick note